sdk/cognitiveservices/azure-cognitiveservices-formrecognizer/DESIGN_DOC.md

```diff
@@ -0,0 +1,436 @@
+# Form Recognizer Design Overview
+
+The Form Recognizer client library provides two clients to interact with the service: `FormRecognizerClient` and 
```

Comment:
I don't think have a strong opinion on this, hoping ux studies might shed some light on discoverability of clients/methods. I like the idea of a `PrebuiltFormClient` for the prebuilt models. If this will grow to many different models/types then it makes sense to split into specific clients, e.g. `ReceiptClient`.

The latest design changes from .NET have the merged `GetAnalyzeResult` outputs so separating custom form clients
might not work here. Before preview we will align client offerings across libraries.

Reply:
How is the service planning on communicating which "feature" of the service that customers should use? And what is the projected future usage ratios (if different than future usage ratios)? 

I think this is a good discussion to have. One thing to call out is that, for client libraries, we generally consider usage as "number of users making the call" whereas if often is "number of service calls" when usage is reported from the service side.

Reply:
1. We expect users to start with our documentation and samples: https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/overview
2. We do not anticipate a change in usage ratios in the immediate future.
3. It's an interesting point, and I don't want to skew our analysis. How can we be sure which is the "right" measurement? I'll work with Kusto to see if we can better understand this, but it would help to set some constraints. E.g. what is a "user"? Is it an Azure Resource/ResourceGroup/Subscription? Should we include UserAgent somehow?